### DISCUSSION TITLE
Manual Installation Instructions

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#89] [Errors]
If the automatic setup fails (sometimes due to proxy/firewalls/etc), here are steps you can take to manually set up:

## Remove previous .kit and .kenv:

please be careful with rm -rf. It recursively deletes directories:

rm -rf ~/.kit
rm -rf ~/.kenv

## Set up the core ".kit":

git clone https://github.com/johnlindquist/kit.git ~/.kit
cd ~/.kit
./install-node.sh --prefix node --platform darwin
./node/bin/npm i

## Set up your environment ".kenv":

git clone https://github.com/johnlindquist/kenv.git ~/.kenv

## Relaunch Kit.app. 
Kit.app will detect the directories exist and skip trying to set them up.

Please post below if you run into issues. I know it's a lot more steps than you probably want to take. Thanks for your patience.

**Comment by filiptammergard** [comment]
Should `./setup/install-node.sh --prefix node --platform darwin` be `./install-node.sh --prefix node --platform darwin`?

Since `install-node.sh` lives in the root.

**Reply by johnlindquist** [reply]
Yes, thank you. I updated the main post to reflect that.

**Comment by mstfgueclue** [comment]
@johnlindquist hey thank you for this great product! 

I was wondering if the manual installation instructions are up to date? I actually have a proxy/firewall problem on a non-personal machine.

**Comment by vivekkodira-pratishthan** [comment]
Would one of you please update these instructions to reflect the current code? The files mentioned in the post don't seem to exist.

**Reply by johnlindquist** [reply]
The best approach now is to install the v3 pre-release 

https://github.com/script-kit/app/releases/tag/v3.19.1

**Reply by vivekkodira-pratishthan** [reply]
Thank you @johnlindquist . Could you please update the instructions here or on the main readme on how to do this? I think they are outdated.

NOTE: This is related to the ongoing issue with MacOS Sequia so I've tried following the original instructions on the repo's readme and in this discussion. I wasn't able to and so am asking for help.
